coffeescripts/content/cards-en.coffee

```diff
@@ -83,89 +88,42 @@ exportObj.translations.English =
         "Lando's Millennium Falcon Expansion Pack": "Lando's Millennium Falcon Expansion Pack"
         "Saw's Renegades Expansion Pack": "Saw's Renegades Expansion Pack"
         "TIE Reaper Expansion Pack": "TIE Reaper Expansion Pack"
+        "ARC-170 Starfighter Expansion": "ARC-170 Starfighter Expansion"
+        "Loose Ships": "Loose Ships"
+        "Fugitives and Collaborators Squadron Pack": "Fugitives and Collaborators Squadron Pack"
+        "First Order Conversion Kit": "First Order Conversion Kit"
+        "M3-A Interceptor Expansion Pack": "M3-A Interceptor Expansion Pack"
     ui:
         shipSelectorPlaceholder: "Select a ship"
         pilotSelectorPlaceholder: "Select a pilot"
-        upgradePlaceholder: (translator, language, slot) ->
-            "No #{translator language, 'slot', slot} Upgrade"
+        upgradePlaceholder: (translator, slot) ->
+            "No #{translator 'slot', slot} Upgrade"
         modificationPlaceholder: "No Modification"
         titlePlaceholder: "No Title"
-        upgradeHeader: (translator, language, slot) ->
-            "#{translator language, 'slot', slot} Upgrade"
+        upgradeHeader: (translator, slot) ->
+            "#{translator 'slot', slot} Upgrade"
         unreleased: "unreleased"
         epic: "epic"
+        "Quickbuild": "Quick Build"
         limited: "limited"
-    byCSSSelector:
-        # Warnings
-        '.unreleased-content-used .translated': 'This squad uses unreleased content!'
-        '.loading-failed-container .translated': 'It appears that you followed a broken link. No squad could be loaded!'
-        '.collection-invalid .translated': 'You cannot field this list with your collection!'
-        '.ship-number-invalid-container .translated': 'A tournament legal squad must contain 2-8 ships!'
-        # Type selector
-        '.game-type-selector option[value="standard"]': 'Extended'
-        '.game-type-selector option[value="hyperspace"]': 'Hyperspace'
-        '.game-type-selector option[value="epic"]': 'Epic'
-        '.game-type-selector option[value="quickbuild"]': 'Quick Build'
-        # Card browser
-        '.xwing-card-browser option[value="name"]': 'Name'
-        '.xwing-card-browser option[value="source"]': 'Source'
-        '.xwing-card-browser option[value="type-by-points"]': 'Type (by Points)'
-        '.xwing-card-browser option[value="type-by-name"]': 'Type (by Name)'
-        '.xwing-card-browser .translate.select-a-card': 'Select a card from the list at the left.'
-        '.xwing-card-browser .translate.sort-cards-by': 'Sort cards by'
-        # Info well
```

Comment:
A little concerned about removing all of the info well translations as well as the buttons and obstacles. Are we replicating them somewhere else that I'm not aware of?

Reply:
Everything that has a `translate` class attribute will also have a `defaultText` attribute. This default text will then be what is translateable in the language file. I hope that I'll replicate all those entries correctly.